recipe/meta.yaml

```diff
@@ -82,22 +68,22 @@ requirements:
 {% set tests_to_skip = "_not_a_real_test" %}
 
 # skip a test that fails with MKL + AVX512 (non-AVX512 passes) (See: https://github.com/scipy/scipy/issues/15533)
-{% set tests_to_skip = tests_to_skip + " or test_x0_equals_Mb[bicgstab]" %} # [blas_impl == 'mkl']
+{% set tests_to_skip = tests_to_skip + " or test_x0_equals_Mb[bicgstab] or test_x0_equals_Mb[bicgstab-nonsymposdef-F]" %} # [blas_impl == 'mkl']
```

Comment:
does it mean that scipy on AVX512 capable CPU in some aspect is broken ? Can we turn it off somehow ?